features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Base/Header/Header.js

```diff
@@ -209,6 +235,28 @@ class Header extends React.Component {
                                         <Link to="/endpoints" style={{color: "#000", textDecoration: 'none'}}>Endpoints</Link>
                                     </MenuItem>
                                 </Menu>
+                                {/* Environment menu */}
```

Comment:
Shall we move this environment menu logic to separate React Component, Because the Header rendering part is already bulky? @chanaka3d please give your comments on this